### PR TITLE
Update MAPI version

### PR DESCRIPTION
**Pull request by sovrasov**
### Summary

Update to fix anomaly saliency map in 1.4 release

### Checklist

- [ ] I have added unit tests to cover my changes.​
- [ ] I have added integration tests to cover my changes.​
- [ ] I have added e2e tests for validation.
- [ ] I have added the description of my changes into CHANGELOG in my target branch (e.g., [CHANGELOG](https://github.com/openvinotoolkit/training_extensions/blob/develop/CHANGELOG.md) in develop).​
- [ ] I have updated the documentation in my target branch accordingly (e.g., [documentation](https://github.com/openvinotoolkit/training_extensions/tree/develop/docs) in develop).
- [ ] I have [linked related issues](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

### License

- [ ] I submit _my code changes_ under the same [Apache License](https://github.com/openvinotoolkit/training_extensions/blob/develop/LICENSE) that covers the project.
      Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below).

```python
# Copyright (C) 2023 Intel Corporation
# SPDX-License-Identifier: Apache-2.0
```

### CODE REVIEW

**Comment by sovrasov** [comment]
OTX version in demo requirements should be updated with the corresponding package version

**Review by ashwinvaidya17 — APPROVED** [review]
Thanks for the efforts!

**Review by chuneuny-emily — APPROVED** [review]

File: src/otx/api/usecases/exportable_code/demo/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
 openvino==2023.0
-openvino-model-api==0.1.6
-otx==1.4.4
+openvino-model-api==0.1.8
+otx @ git+https://github.com/openvinotoolkit/training_extensions/@4abe2ea89680d4bdf54f97e1fa78abebd65c7e36#egg=otx
```
Comment (goodsong81) [inline src/otx/api/usecases/exportable_code/demo/requirements.txt:3]: @yunchu Would you update this line when you release a new package?

**Review by goodsong81 — APPROVED** [review]
LGTM. Left a comment not to forget.

**Pull request merged**